integration/samples/package-js/secondary/ui-lib.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+
+import { BazComponent } from './baz/baz.component';
+
+@NgModule({
+  declarations: [
+    BazComponent,
+  ],
+  exports: [
+    BazComponent,
+  ]
+})
+export class UiLibModule {}
```

Comment:
Think we should keep the sample really minimalistic as it's just there to verify the conf location... is it ok I'll reduce it after merging?

Reply:
Yep absolutely